datastream-server-restli/src/main/java/com/linkedin/datastream/server/dms/DatastreamStore.java

```diff
@@ -66,4 +66,10 @@ void updatePartitionAssignments(String key, Datastream datastream, HostTargetAss
    * @param key Name of the original datastream whose numTasks znode is to be deleted
    */
   void deleteDatastreamNumTasks(String key);
+
+  /**
+   * Delete the assignment tokens (if any) associated with the provided key
+   * @param key Name of the datastream whose assignment tokens have to be deleted
+   */
+  void deleteAssignmentTokens(String key);
```

Comment:
You are leaking implementation details in the DatastreamStore interface. Can you think of another generic name?

Reply:
I cant think of a better name. I'm open to suggestions.

Reply:
Something related to forcing datastream stop maybe..

Reply:
renamed it to `forceCleanupDatastream`. Not particularly fond of it, but slightly better